airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -66,6 +208,93 @@ public JsonNode toJdbcConfig(JsonNode config) {
     return Jsons.jsonNode(configBuilder.build());
   }
 
+  private static boolean isCdc(JsonNode config) {
```

Comment:
I usually prefer separating stuff out. I think in this case returning the inline variable is better.

So
```
    return config.hasNonNull("replication_method")
        && ReplicationMethod.valueOf(config.get("replication_method").asText())
            .equals(ReplicationMethod.CDC);
```

This is a nit.